style/styles.css

```
@@ -746,24 +507,26 @@ h3 {
-@media screen and (max-width: 640px) {
-  .flex-second {
-    order: 2
-  }
+#footer:after {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 200px;
+  background: -webkit-linear-gradient(0deg, rgb(29 33 44 / 0%) 17%, rgb(29 33 44 / 100%) 100%);
```

Stylelint (17.3.0)

[error] Unexpected vendor-prefixed value "-webkit-linear-gradient" (value-no-vendor-prefix)

(value-no-vendor-prefix)